src/Router.tsx

```diff
@@ -13,23 +15,51 @@ const ImportImageWizard = lazy(
   () => import('./Components/CreateImageWizard/ImportImageWizard')
 );
 const CreateImageWizard = lazy(() => import('./Components/CreateImageWizard'));
+const isOnPremise = process.env.APP_TYPE_ENV;
```

Comment:
Let's wrap this in a utility function in `src/Utilities/useGetEnvironment` as well?